edgexfoundry/app-service-configurable
# refactor!: Adjust configuration for reworked MessageBus config

closes #489

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
Build service locally
Run with none-secure stack for each profile.
Verify each profile that uses the messagebus starts and runs as expected (i.e. connects to MessageBus)
Verify each profile that doesn't use the messagebus starts and runs as expected (i.e. w/o connecting to MessageBus)


## New Dependency Instructions (If applicable)
